### ISSUE TITLE
Saving kapacitor alerts back to InfluxDB

### ISSUE THREAD

**Issue by lpalm** [influxdata/kapacitor#286]
According to the documentation, there's not InfluxDBOut node for alerts. It's unclear if the HTTP post output mode can be easily linked to InfluxDB's HTTP API. However, It would be super handy to be able to put alerts into InfluxDB so that it can be displayed in Grafana (or Chronograf depending on how it matures) as [annotations](http://docs.grafana.org/reference/annotations/) and/or in a [table](http://docs.grafana.org/reference/table_panel/). The alert datapoints could include the message as a field and the alert ID as a tag or measurement name.

The general goal here is to have a tickscript produce a graph of the alert input(s), along with annotations for every time the alert fires. Both so we can look back on previous alerts in an intuitive way (how were they clustered? How did the inputs look like before the alert fired?) and so we can get a clear view of when new alerts would have fired during backtesting.

**Comment by nathanielc** [comment]
Yep, see https://github.com/influxdata/kapacitor/issues/93. We just need to have the alert node send its alert info down to child nodes.

**Comment by sslupsky** [comment]
This would be useful.  As a work around, you could probably use logstash to parse the alert log and put the data into influx or elasticsearch.

**Comment by framiere** [comment]
I agree this would be a neat feature that would simplify separation of concerns. 
I currently use logstash to retrieve the alerts and ship them back to influx (via kafka), it's quite cumbersome.

Please note that the output should be ideally linked to #408 

Before 
```
                 +---------------------+
                 |                     |
                 |                     |
         +---->  |                     |
         |       |    Windows/Linux    |
         |       |                     |
Collect  |       |                     |
         |       |                     |
         |       |                     |
         |       +---------------------+
         +-------+                     |   Ship metrics to kafka
         +-------+      Telegraf       +-------------->
Forward  |       |                     |
metrics  |       +---------------------+
         +-----> |                     |
         +-------+      Kapacitor      |
         |       |                     |
Alert    |       +---------------------+
         +-----> |                     |
                 |      Logfiles       |
         +-----> |                     |
Read     |       +---------------------+   
         +-------+                     |   Ship alert to kafka
                 |      Logstash       +-------------->
                 |                     |   
                 +---------------------+
```

After

```
                 +---------------------+
                 |                     |
                 |                     |
         +---->  |                     |
         |       |    Windows/Linux    |
         |       |                     |
Collect  |       |                     |
         |       |                     |
         |       |                     |
         |       +---------------------+
         +-------+                     |   Ship metrics to kafka
         +-------+      Telegraf       +-------------->
Forward  |       |                     |
metrics  |       +---------------------+
         +-----> |                     |   Ship alert to kafka
                 |      Kapacitor      +-------------->
                 |                     |
                 +---------------------+
```

So much simpler to document and reason about ! 
Added bonus : we can remove logstash dependency/overhead.

**Comment by nathanielc** [comment]
Closed with #465
